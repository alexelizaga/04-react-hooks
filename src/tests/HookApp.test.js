import renderer from 'react-test-renderer';

import { HookApp } from '../HookApp';

describe('Test HookApp', () => {

  test('should render <HookApp /> correctly', () => {
    const tree = renderer
      .create(<HookApp />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});