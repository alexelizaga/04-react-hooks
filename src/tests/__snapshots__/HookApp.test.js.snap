// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HookApp should render <HookApp /> correctly 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
